environments/common/ecr/ecr.tf

```
@@ -0,0 +1,22 @@
+resource "aws_kms_key" "ecr_kms" {
+  description         = "ECR KMS key"
+  enable_key_rotation = true
+}
+
+
+# tfsec:ignore:aws-ecr-enforce-immutable-repository
+resource "aws_ecr_repository" "trade-tariff-ecr_repo" {
```

```suggestion
resource "aws_ecr_repository" "trade_tariff_ecr_repo" {
```

Minor request - can you change these to underscores? For consistency with other resources 😄